.github/workflows/cron.yml

```diff
@@ -0,0 +1,61 @@
+name: Cron
+
+on:
+  schedule:
+    # every friday at 10:00 am
+    - cron: '0 10 * * 5'
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust:
+          - 1.46.0
```

Comment:
I think it is below MSRV from README, was it intentional?

Reply:
I guess the MSRV was updated after this PR was created. I will update this PR. Thanks for pinging me!